paparazzi-gradle-plugin/src/test/projects/multiplatform-plugin-with-android/gradle.properties

```diff
@@ -1 +1,2 @@
 kotlin.mpp.androidSourceSetLayoutVersion1.nowarn=true
+android.useAndroidX=true
```

Comment:
The multiplatform test started to fail because `TestRuntimeClasspath` contained androidx dependencies, and `useAndroidX` was not set to true